<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_webcam">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>353</width>
     <height>563</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/try-this-trainer-approved-chest-day-workout.jpg);
border-radius: 18px; 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_profile">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>840</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-image: url(:/images/User.jpg);

border-radius: 5px; 
padding: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_weighlifting">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>150</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
            background-color: #EFEFEF;
            color: black;              
            border-radius: 5px; 
            padding: 5px;
}
QPushButton:hover {
            background-color: #EAEAEA; 
}
QPushButton:pressed {
            background-color: #EAEAEA;
}</string>
   </property>
   <property name="text">
    <string>WEIGHTLEFTING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cardio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>150</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
            background-color: #EFEFEF;
            color: black;              
            border-radius: 5px; 
            padding: 5px;
}
QPushButton:hover {
            background-color: #EAEAEA; 
}
QPushButton:pressed {
            background-color: #EAEAEA;
}</string>
   </property>
   <property name="text">
    <string>CARDIO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_camera">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>830</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-image: url(:/images/VideoStabilization.png);
border: 1px solid #2E7D32;
border-radius: 5px; 
padding: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/images/logo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_home">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>840</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/home.png);
border-radius: 5px; 
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>393</width>
     <height>852</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FAFAFA;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>821</y>
     <width>393</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 79, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>btn_camera</zorder>
  <zorder>lb_webcam</zorder>
  <zorder>btn_profile</zorder>
  <zorder>btn_weighlifting</zorder>
  <zorder>btn_cardio</zorder>
  <zorder>lb_logo</zorder>
  <zorder>btn_home</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
